0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -123,6 +124,11 @@ Unsubscribe from ledger events.
 
 Query whether the plugin is currently connected.
 
+#### isOpen
+<code>ledgerPlugin.isOpen() ⇒ Boolean</code>
+
+Query whether the plugin is currently connected and has an active websocket.
```

Comment:
They are different. Example:

* Initially: `connected=false` and `open=false`.
* When `plugin.connect()` is called successfully, `connected=true` and `open=true`
* If some time later the websocket connection drops, `connection=true` but `open=false`.
* The plugin will try to reestablish the websocket, and when it succeeds, `connection=true` and `open=true`.
* When `disconnect()` if called, `connection=false` and `open=false`.

`isConnected()` is used to check that functions like `getPrefix()` and `getAccount()` can be called, because the first `connect()` call sets up the urls & metadata that the plugin needs.

`isOpen()` will be used by the connector for https://github.com/interledgerjs/ilp-connector/issues/248 -- i.e. to allow the connector to refuse to quote or settle payments without an _active_ websocket.

Perhaps the names should be changed to reduce confusion? `s/isConnected/isReady` and `s/isOpen/isConnected`, or similar. Though, that would be a breaking change.